From cf4300ce5a13f074acf3decdb00e1e2715065378 Mon Sep 17 00:00:00 2001
From: Yinbo Zhu <yinbo.zhu@nxp.com>
Date: Wed, 5 Dec 2018 14:27:20 +0800
Subject: [PATCH 385/706] arm64: dts: ls1028a: Add quirk for Erratum A009116

Add "snps,quirk-frame-length-adjustment" property to USB3 node for
erratum A009116.  This property provides value of GFLADJ_30MHZ for post
silicon frame length adjustment.

Signed-off-by: Yinbo Zhu <yinbo.zhu@nxp.com>
(cherry picked from commit cfcc97bd2b129f9ad609810635c0f0845f361ad2)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi b/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi
index 8fd15bfab528..c0eb89a70e9b 100755
--- a/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi
@@ -367,6 +367,7 @@
 			interrupts= <0 80 0x4>;
 			dr_mode= "host";
 			snps,dis_rxdet_inp3_quirk;
+			snps,quirk-frame-length-adjustment = <0x20>;
 			snps,incr-burst-type-adjustment = <1>, <4>, <8>, <16>;
 		};
 
@@ -376,6 +377,7 @@
 			interrupts= <0 81 0x4>;
 			dr_mode= "host";
 			snps,dis_rxdet_inp3_quirk;
+			snps,quirk-frame-length-adjustment = <0x20>;
 			snps,incr-burst-type-adjustment = <1>, <4>, <8>, <16>;
 		};
 
-- 
2.17.1

